iroh-net/src/magic_endpoint.rs

```diff
@@ -134,7 +148,7 @@ impl MagicEndpointBuilder {
             keypair,
             bind_port,
             Some(server_config),
-            self.derp_map,
+            self.derp_map.unwrap_or_default(),
```

Comment:
I am not sure how the behaviour was before, but I think we should make the default use the default derp map, which I don't think is the case now, as the default is `None`, or an empty derpmap